src/components/logList.tsx

```diff
@@ -3,17 +3,15 @@ import styles from "@/styles/List.module.css";
 
 const logList = () => {
   const item = [
-    { date: "2023 / 12 / 23 (土) ", title: "クリスマス飲み会" },
-    { date: "2023 / 12 / 31 (日) ", title: "忘年会" },
+    { id: "1", date: "2023/12/23(土)", title: "クリスマス飲み会" },
+    { id: "2", date: "2023/12/31(日)", title: "忘年会" },
```

Comment:
```suggestion
    { id: "1", date: 1703257200, title: "クリスマス飲み会" },
    { id: "2", date: 1703948400, title: "忘年会" },
```
日付管理はunix timestampを使ったほうが便利です